README.md

```diff
@@ -302,6 +302,10 @@ Technoogy Icon | Technoogy Name | URL
 :-----:|  :-----: | -----
 <img height="50" src="https://user-images.githubusercontent.com/25181517/184103699-d1b83c07-2d83-4d99-9a1e-83bd89e08117.png"> | Selenium | `https://user-images.githubusercontent.com/25181517/184103699-d1b83c07-2d83-4d99-9a1e-83bd89e08117.png`
 <img height="50" src="https://user-images.githubusercontent.com/25181517/184117353-4b437677-c4bb-4f4c-b448-af4920576732.png"> | Cucumber | `https://user-images.githubusercontent.com/25181517/184117353-4b437677-c4bb-4f4c-b448-af4920576732.png`
+<img height="50" src="https://user-images.githubusercontent.com/68279555/200387386-276c709f-380b-46cc-81fd-f292985927a8.png"> | Cypress | `https://user-images.githubusercontent.com/68279555/200387386-276c709f-380b-46cc-81fd-f292985927a8.png`
+<img height="50" src="https://user-images.githubusercontent.com/68279555/200387382-944dbebe-5814-4b0e-b89b-86c1dda0bf97.png"> | Chai | `https://user-images.githubusercontent.com/68279555/200387382-944dbebe-5814-4b0e-b89b-86c1dda0bf97.png`
```

Comment:
@joaopaulomoreira Hello, thank you so much for contribiution !!!! You are the best !

We have to change this image becase it is 450px x 400px and should be 400x400 because other icons can then display not correct

Reply:
fixed